canonical-web-and-design/snapcraft.io
# Remove unused content of package step of first snap flow

With introduction of interactive snapcraft.yaml viewer we are not showing these steps on package page of first snap flow.
This PR removes this content to avoid confusion and unnecessary maintenance.

### QA
- ./run or demo
- make sure package step of first snap flow is displayed correctly
